### PR TITLE
update graphql servlet dependency

### PR DESCRIPTION
**Pull request by jmccaull**
*updated graphql servlet version to 7.3
*moved servlet config into a conditional on missing bean
*autowired new GraphQLBatchExecutionHandlerFactory

### CODE REVIEW

**Comment by jmccaull** [comment]
@oliemansm that was quick, thanks! Could we likewise get a release for this when you have time please?

File: graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLWebAutoConfiguration.java [hunk 1]
```diff
@@ -126,6 +127,9 @@
     @Autowired(required = false)
     private MultipartConfigElement multipartConfigElement;
 
+    @Autowired(required = false)
+    private GraphQLBatchExecutionHandlerFactory graphQLBatchExecutionHandlerFactory;
```
Comment (hanjb) [inline graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLWebAutoConfiguration.java]: what happens if GraphQLBatchExecutionHandlerFactory is not injected? can batched query still be processed ?

Comment (jmccaull) [reply graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLWebAutoConfiguration.java]: yes, there is a default in the config object and it is used if none/null is passed in. Default is previous behavior

**Comment by oliemansm** [comment]
@jmccaull Releasing version 5.6.0 as we speak.

**Pull request merged**